app/react/src/client/preview/init.js

```diff
@@ -1,11 +1,11 @@
 import keyEvents from '@storybook/ui/dist/libs/key_events';
-import { selectStory } from './actions';
+import { Actions } from '@storybook/core/client';
```

Comment:
Would exporting & importing all actions individually be better here?

Reply:
I chose to bundle them to keep the namespace smaller. Open to counter-arguments.